types/soap/index.d.ts

```diff
@@ -1,87 +1,188 @@
-// Type definitions for soap 0.18
+// Type definitions for soap 0.19
 // Project: https://www.npmjs.com/package/soap
-// Definitions by: Leo Liang <https://github.com/aleung>, Cage Fox <https://github.com/cagefox>
+// Definitions by:  Leo Liang <https://github.com/aleung>
+//                  Cage Fox <https://github.com/cagefox>
+//                  Onur Yildirim <https://github.com/onury>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
-import * as events from 'events';
+import { EventEmitter } from 'events';
 
-interface Security {
+export interface ISOAPMethod {
```

Comment:
The `soap` module itself has the followings:
- `WSSecurity` - not `WsSecurity`
- `ClientSSLSecurity` - not `ClientSslSecurity`
- `ClientSSLSecurityPFX` - not `ClientSslSecurityPfx`

Also, the original module exports `WSDL` not `Wsdl`

I'm not only following my preferences, this is the original module's style/convention.